House-of-glass/src/components/Product List/ProductList.jsx

```diff
@@ -1,26 +1,25 @@
-import React from 'react'
-import VerticalCard from 'components/Cards/Vertical/VerticalCard';
-import Filters from 'components/Filters/Filters';
-import { useFilter } from 'components/Filters/FilterContext';
-import { products } from 'backend/db/products';
+import React from "react";
+import Filters from "components/Filters/Filters";
+import { useProduct } from "components/Filters/ProductContext";
+import ProductCard from "components/Cards/Vertical/ProductCard";
 
 export default function ProductList() {
-
-    const {finalProductList} = useFilter();
-
-
+  const { finalProductList } = useProduct();
```

Comment:
Just `products` or `productList` is enough

Reply:
Thank You Simran. Updated in the latest commit.